libp2p/protocols/mix/mix_protocol.nim

```diff
@@ -249,6 +271,21 @@ proc handleMixNodeConnection(
   of InvalidMAC:
     mix_messages_error.inc(labelValues = ["Intermediate/Exit", "INVALID_MAC"])
 
+proc handleMixNodeConnection(
+    mixProto: MixProtocol, conn: Connection
+) {.async: (raises: [LPStreamError, CancelledError]).} =
+  defer:
+    await conn.close()
```

Comment:
if this connections got error and we created new connection. then old connection is closed here, but new connection will never be closed?